src/main/default/lwc/lookup/lookup.html

```diff
@@ -146,13 +148,13 @@
             </template>
             <!-- Multi-selection end -->
 
-            <!-- Errors start -->
-            <template for:each={errors} for:item="error">
-                <label key={error.id} role="alert" class="slds-form-element__label slds-var-m-top_xx-small form-error"
-                    >{error.message}</label
-                >
+            <!-- Error start -->
```

Comment:
I would not change how error is displayed. Before you could pass in array and I would not change that because it can happen we have more errors.